src/components/table/tableContainer/styles.ts

```diff
@@ -0,0 +1,6 @@
+import { styled } from '@linaria/react'
+
+export const ElTableContainer = styled.div`
+  gap: 0px;
+  background: var(--fill-white);
```

Comment:
**question:** Is this strictly needed? Do we care what the background of this container is?

Reply:
**answer:** I copied this across from Figma, I'll double-check with Andrei if it's a requirement. If not, I will remove it.